projects/plugins/jetpack/modules/stats/class-jetpack-stats-dashboard-widget.php

```diff
@@ -0,0 +1,157 @@
+		if ( has_action( 'jetpack_dashboard_widget' ) ) {
+			$jetpack_logo = new Jetpack_Logo();
+			$widget_title = sprintf(
+				// translators: Placeholder is a Jetpack logo.
+				__( 'Stats by %s', 'jetpack' ),
+				$jetpack_logo->get_jp_emblem( true )
+			);
+
+			// Wrap title in span so Logo can be properly styled.
+			$widget_title = sprintf(
+				'<span>%s</span>',
+				$widget_title
+			);
+
+			wp_add_dashboard_widget(
+				'jetpack_summary_widget',
+				$widget_title,
+				array( __CLASS__, 'dashboard_widget' )
+			);
+			wp_enqueue_style( 'jetpack-dashboard-widget', plugins_url( 'css/dashboard-widget.css', JETPACK__PLUGIN_FILE ), array(), JETPACK__VERSION );
+			wp_style_add_data( 'jetpack-dashboard-widget', 'rtl', 'replace' );
+
+			// If we're inactive and not in offline mode, sort our box to the top.
+			if ( ! Jetpack::is_connection_ready() && ! ( new Status() )->is_offline_mode() ) {
+				global $wp_meta_boxes;
+
+				$dashboard = $wp_meta_boxes['dashboard']['normal']['core'];
+				$ours      = array( 'jetpack_summary_widget' => $dashboard['jetpack_summary_widget'] );
+
+				$wp_meta_boxes['dashboard']['normal']['core'] = array_merge( $ours, $dashboard );
+			}
+		}
+	}
+
+	/**
+	 * Fires when the dashboard is loaded.
+	 *
+	 * @since 3.4.0
+	 */
+	public static function dashboard_widget() {
+		do_action( 'jetpack_dashboard_widget' );
+	}
+
+	/**
+	 * Load the widget footer showing Akismet stats.
+	 */
+	public static function dashboard_widget_footer() {
+		?>
+		<footer>
+
+		<div class="protect">
+			<h3><?php esc_html_e( 'Brute force attack protection', 'jetpack' ); ?></h3>
+			<?php if ( Jetpack::is_module_active( 'protect' ) ) : ?>
+				<p class="blocked-count">
+					<?php echo number_format_i18n( get_site_option( 'jetpack_protect_blocked_attempts', 0 ) ); ?>
+				</p>
+				<p><?php echo esc_html_x( 'Blocked malicious login attempts', '{#} Blocked malicious login attempts -- number is on a prior line, text is a caption.', 'jetpack' ); ?></p>
+			<?php elseif ( current_user_can( 'jetpack_activate_modules' ) && ! ( new Status() )->is_offline_mode() ) : ?>
+				<a href="
+				<?php
+				echo esc_url(
+					wp_nonce_url(
+						Jetpack::admin_url(
+							array(
+								'action' => 'activate',
+								'module' => 'protect',
+							)
+						),
+						'jetpack_activate-protect'
+					)
+				);
+				?>
+							" class="button button-jetpack" title="<?php esc_attr_e( 'Protect helps to keep you secure from brute-force login attacks.', 'jetpack' ); ?>">
+					<?php esc_html_e( 'Activate brute force attack protection', 'jetpack' ); ?>
+				</a>
+			<?php else : ?>
+				<?php esc_html_e( 'Brute force attack protection is inactive.', 'jetpack' ); ?>
+			<?php endif; ?>
+		</div>
+
+		<div class="akismet">
+			<h3><?php esc_html_e( 'Anti-spam', 'jetpack' ); ?></h3>
+			<?php if ( is_plugin_active( 'akismet/akismet.php' ) ) : ?>
+				<p class="blocked-count">
+					<?php echo number_format_i18n( get_option( 'akismet_spam_count', 0 ) ); ?>
```

Comment:
Gotta figure out how best to escape this.

Reply:
`esc_html` be fine? We're expecting just a number right?

Reply:
D'oh, you're right, I was trying `esc_html_e` but this obviously doesn't need to be translated.